lib/msal-browser/docs/request-response-object.md

```diff
@@ -0,0 +1,128 @@
+const resp = msalInstance.loginPopup(interactiveRequest);
+const resp2 = msalInstance.acquireTokenPopup(interactiveRequest);
+```
+
+- TokenRenewParameters - used for `acquireTokenSilent()`
+
+```javascript
+const silentRequest = {
+    scopes: [],
+    resource: "",
+    extraQueryParameters: {},
+    authority: "",
+    correlationId: "",
+    account: null,
+    forceRefresh: false
+};
+
+const resp = msalInstance.acquireTokenSilent(silentRequest);
+```
+
+### Interactive request options
+
+| Option | Description | Format | 
+| ------ | ----------- | ------ |
+| `scopes` | See [below](#scopes). | String array |
+| `resource` | URI of the resource to acquire access for. | String |
+| `extraQueryParameters` | `key`:`value` object with additional parameters to be sent as part of URI request. For example, `extraQueryParameters: {k1: v1}` is included in the request URI as `?k1=v1`. | String dictionary - `{key: value}`|
+| `authority` | URI of the tenant to authenticate and authorize with. Usually takes the form of `https://{uri}/{tenantid}`. Overwrites the authority set in the [`PublicClientApplication` configuration object](./configuration.md). | String in URI format with tenant - `https://{uri}/{tenantid}` |
+| `correlationId` | ID included in log messages and requests to trace executions. Usually a UUID/GUID. | String |
+| `extraScopesToConsent` | Additional scopes to pre-consent to as part of login call, allowing you to make silent calls for additional resources without interaction. | String array |
+| `prompt` | A string value to control the behavior seen by the user. See [below](#prompt-values) for more info | String with one of the following values: `"login"`, `"select_account"`, `"consent"`, `"none"`. |
+| `claimsRequest` | A stringified JSON object containing requested claims for a token. | Stringified JSON object |
+| `userRequestState` | OAuth `state` parameter sent in the authorization code request and returned by the service unchanged in the response. You can use this parameter to restore app state on a response. | String |
+| `account` | User account object used to perform non-interactive sign-in. Must be an object of type `Account` as defined by MSAL. This account object is returned in the [response object](#response) and by the `getAccount()` API. | [Account.ts](../../msal-common/src/auth/Account.ts) |
+| `sid` | Session id of the user to identify an active session and perform non-interactive sign-in. Usually part of id token claims. | String |
+| `login_hint` | UPN or `preferred_username` of user account attempting to retrieve a token. Allows for non-interactive sign-in. | String |
+
+### Silent request options
+All descriptions of silent request options can be found above except for:
+
+| Option | Description | Format | 
+| ------ | ----------- | ------ |
+| `forceRefresh` | If true, `acquireTokenSilent` will not returned the cached tokens and will use the refresh token to retrieve a new set of tokens. | boolean |
+
+### Scopes
```

Comment:
not needed right now, but @jasonnutter we can review the `scopes` info here from the POV of redirecting all `scopes` usage related questions here.

Reply:
+1, it would be good to start having repo docs for all usage related questions.